internal/types/metadata_test.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestMaintainer(t *testing.T) {
+	assert.Check(t, is.Equal(Maintainer{"foo", "foo@bar.com"}.String(), "foo <foo@bar.com>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", ""}.String(), "foo <>"))
+	assert.Check(t, is.Equal(Maintainer{"foo", "bar"}.String(), "foo <bar>"))
+	assert.Check(t, is.Equal(Maintainer{"", ""}.String(), " <>"))
```

Comment:
We should error on this one. I think the metadata validation can handle this case.

Reply:
yep, but we don't right now :stuck_out_tongue: